pom.xml

```diff
@@ -143,21 +148,23 @@
         <dependency><!-- exists in the core -->
             <groupId>org.jenkins-ci.modules</groupId>
             <artifactId>instance-identity</artifactId>
-            <version>1.3</version>
+            <version>2.1</version><!-- matches 2.60.3 -->
```

Comment:
So is it provided by core?

Reply:
That's a good question, it was a previous annotation.  is there a good way to verify?

Reply:
I actually just downloaded the core.jar file and it is not included, so I'm going to remove this comment & provided

Reply:
i remind that it's jar and not a plugin...

Reply:
Same idea as my previous comment. It's pulled by the core indeed.

See https://github.com/jenkinsci/jenkins/blob/cc964ef664250b6737e471ef546a46a37421cb4f/war/pom.xml#L95-L99